src/components/Stack/Stack.tsx

```diff
@@ -38,7 +46,7 @@ export const Stack = forwardRef<HTMLElement, StackProps>((props, ref) => {
         {...boxProps}
         ref={ref}
         style={{
-          gap: units ? units * 0.5 * SPACE_UNIT : undefined,
+          gap: gap ? gap * 0.5 : undefined,
```

Comment:
for more flexibility the gap prop should be only set by outside, and not multiplied by `0.5`

Reply:
this bit handle the divider case is seems strange and misleading to me to have it behave in 2 different ways if a divider is present or not. 
With the present implementation `gap` define the space between childs in both cases, IF a divider is present the divider is placed in the middle of this space, this is the reason for this calc

Reply:
got the reason but maybe is more misleading is to set a gap size and find an half gap size with a divider in between. Maybe the divider itself could override the gap.

However it's fine for me to keep this kind of opinionated implementation, if you want to keep please add a jsdoc to explain the `gap` behaviour